core/common/src/main/java/alluxio/underfs/UfsStatus.java

```diff
@@ -38,13 +39,16 @@
    * @param owner of the file
    * @param group of the file
    * @param mode of the file
+   * @param lastModifiedTimeMs UTC time
```

Comment:
Isn't this epoch time?

Reply:
Yeah I just moved the javadoc from UfsFileStatus. I will update it.